### ISSUE TITLE
pod can't be installed

### ISSUE THREAD

**Issue by AbdulazizAlmohsen** [kysomers/CheckboxList#2]
I can't install your pod. Is it because your version ? 
```

# Uncomment this line to define a global platform for your project
# platform :ios, '9.0'

target 'Jaee2' do
  # Comment this line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for Jaee2
pod 'Haneke', '~> 1.0'
pod "TextFieldEffects"
pod 'STLocationRequest'
pod 'SwiftyJSON'
pod 'Alamofire', '~> 4.4'
pod 'PasswordTextField'
pod 'OneSignal', '>= 2.5.2', '< 3.0'
pod 'Spring', :git => 'https://github.com/MengTo/Spring.git', :branch => 'swift3'
pod 'HMScrollNavigationBar'
pod 'DynamicButton', '~> 4.0.0'
pod 'CheckboxListSwift'

end
```

>Analyzing dependencies
[!] Unable to find a specification for `CheckboxListSwift`

**Comment by kysomers** [comment]
Oh shoot, I know what caused that. Sorry, I'll get it fixed in the next few days.

**Comment by kysomers** [comment]
Fixed and updated to Swift 3.1 and iOS 10.0. Thank you for helping me out by letting me know about the issue! Cheers!
